releases/release-5.0.6.md

```diff
@@ -0,0 +1,168 @@
+
+Release date: December 28, 2021
+
+TiDB version: 5.0.6
+
+## Compatibility changes
+
++ Tools
+
+    + TiCDC
+
+        - Output the cdc server cmd error from stdout to stderr. [#3133](https://github.com/pingcap/tiflow/issues/3133)
+
+## Improvements
+
++ TiDB
+
+    (dup) - Fix the issue that optimistic transaction conflicts might cause transactions to block each other [#11148](https://github.com/tikv/tikv/issues/11148)
+    (dup) - Fix the issue of false positive error log `invalid cop task execution summaries length` for MPP queries [#1791](https://github.com/pingcap/tics/issues/1791)
+    (dup) - Show the affected SQL statements in the debug log when the coprocessor encounters a lock, which is helpful in diagnosing problems [#27718](https://github.com/pingcap/tidb/issues/27718)
+
++ TiKV
+
+    - Move verify_checksum to import-thread from apply-thread. [#11239](https://github.com/tikv/tikv/issues/11239)
+    - Add metrics for raft log garbage-collect to locate performance problem. [#11374](https://github.com/tikv/tikv/issues/11374)
+    - Hide untouched storage commands' metrics in grafana dashboard [#11681](https://github.com/tikv/tikv/issues/11681)
+
++ PD
+
+    (dup) - Speed up the exit process of schedulers [#4146](https://github.com/tikv/pd/issues/4146)
+    - Make scatter range scheduler work better by allowing empty region schedule and fix config [#4497](https://github.com/tikv/pd/issues/4497)
+
++ Tools
+
+    + TiCDC
+        (dup) - Optimize rate limiting control on TiKV reloads to reduce gPRC congestion during changefeed initialization [#3110](https://github.com/pingcap/ticdc/issues/3110)
+        - Reduce lock competition in sink module. [#2760](https://github.com/pingcap/tiflow/pull/2760)
+        (dup) - Fix the issue that scanning stock data might fail due to TiKV performing GC when scanning stock data takes too long [#2470](https://github.com/pingcap/tiflow/issues/2470)
+        (dup) - Fix the issue that changefeed does not fail fast enough when the ErrGCTTLExceeded error occurs [#3111](https://github.com/pingcap/ticdc/issues/3111)
+        (dup) - Add a tick frequency limit to EtcdWorker to prevent frequent etcd writes from affecting PD services [#3112](https://github.com/pingcap/ticdc/issues/3112)
+        - Support batch messages to reduce EtcdWorker tick. [3112](https://github.com/pingcap/tiflow/issues/3112)
+        (dup) - Fix OOM in container environments [#1798](https://github.com/pingcap/ticdc/issues/1798)
+        - Add Kafka sink default configuration config.Metadata.Timeout. [#3352](https://github.com/pingcap/tiflow/issues/3352)
+        - Change Kafka sink default `MaxMessageBytes` to 1MB. [#3081](https://github.com/pingcap/tiflow/issues/3081)
+        - Add more monitor metric and alert, including "no owner alert" [#4054](https://github.com/pingcap/tiflow/issues/4054), "mounter row" , "table sink total row" , "buffer sink total row" [#1606](https://github.com/pingcap/tiflow/issues/1606)
```

Comment:
```suggestion
    + TiCDC
        (dup) - Optimize rate limiting control on TiKV reloads to reduce gPRC congestion during changefeed initialization [#3110](https://github.com/pingcap/ticdc/issues/3110)
        - Reduce lock competition in sink module. [#2760](https://github.com/pingcap/tiflow/pull/2760)
        (dup) - Fix the issue that scanning stock data might fail due to TiKV performing GC when scanning stock data takes too long [#2470](https://github.com/pingcap/tiflow/issues/2470)
        (dup) - Fix the issue that changefeed does not fail fast enough when the ErrGCTTLExceeded error occurs [#3111](https://github.com/pingcap/ticdc/issues/3111)
        (dup) - Add a tick frequency limit to EtcdWorker to prevent frequent etcd writes from affecting PD services [#3112](https://github.com/pingcap/ticdc/issues/3112)
        - Support batch messages to reduce the EtcdWorker tick frequency. [#3112](https://github.com/pingcap/tiflow/issues/3112)
        (dup) - Fix OOM in container environments [#1798](https://github.com/pingcap/ticdc/issues/1798)
        - Add the default configuration for metadata timeout `config.Metadata.Timeout` in Kafka sink. [#3352](https://github.com/pingcap/tiflow/issues/3352)
        - Change the default value of `MaxMessageBytes` in Kafka sink to 1MB. [#3081](https://github.com/pingcap/tiflow/issues/3081)
        - Add more monitor metrics and alerts, including "no owner alert" [#4054](https://github.com/pingcap/tiflow/issues/4054), "mounter row" , "table sink total row" , "buffer sink total row" [#1606](https://github.com/pingcap/tiflow/issues/1606)
```